typescript-arrays/main.ts

```diff
@@ -0,0 +1,26 @@
+const colors: any[] = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+
+const america: string = `${colors[0]}, ${colors[1]}, ${colors[2]}`;
+console.log('America is', america);
+
+colors[2] = 'green';
+
+const mexico: string = `${colors[0]}, ${colors[1]}, ${colors[2]}`;
+console.log('Mexico is', mexico);
+
+console.log('value of colors:', colors);
+console.log('typeof colors:', typeof colors);
+
+const students: any[] = ['Frank', 'Bob', 'Chris', 'David'];
+
+const numberOfStudents: number = students.length;
+console.log(`There are ${numberOfStudents} students in the class`);
+const lastIndex: number = numberOfStudents - 1;
+const lastStudent: number = students[lastIndex];
```

Comment:
The type here should have been an error. You are taking students[lastIndex] which is a student's name and assigning it to a variable that is a number type. lastStudent should be the same type as the student's name.